### ISSUE TITLE
ua-parser-js Vulnerability

### ISSUE THREAD

**Issue by neryb** [yusinto/ld-redux#46]
Hello, our team uses ld-redux and a Whitesource has flagged that the version of ua-parser-js being used is marked as a high severity vulnerability.

Info:
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7733

Resolution: Bump version to 0.7.22
https://github.com/yusinto/ld-redux/blob/74f41923372c35e5b0e7c4607444103fb3e73e11/package.json#L57

**Comment by JBallin** [comment]
@yusinto is this on your radar? Thanks

**Comment by JBallin** [comment]
@yusinto the current recommended version is `0.7.30` - which you can see [fbjs just updated to](https://github.com/facebook/fbjs/commit/5893dd1ee1c784891df4cd2519c586a9da46c8a8).

Would this be resolved faster if we opened a PR ourselves? Let us know how we can help - thank you!

https://snyk.io/vuln/npm:ua-parser-js

**Comment by yusinto** [comment]
Yes if someone could submit a PR that will be much appreciated. Thank you for your help.

**Comment by neryb** [comment]
Closing this issue as the original vulnerability we opened this issue with was addressed due to the use of the caret in the package version.
